x-pack/test/accessibility/apps/search_profiler.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'security']);
+  const testSubjects = getService('testSubjects');
+  const aceEditor = getService('aceEditor');
+  const a11y = getService('a11y');
+  const flyout = getService('flyout');
+
+  // this test is skipped due to https://github.com/elastic/kibana/issues/62531
```

Comment:
Looks like this issue is closed. Can we enable the tests?

Reply:
yes. re-enabled.